src/tir/schedule/primitive/get_block_loop.cc

```diff
@@ -238,7 +238,7 @@ struct GetOutputBlocksTraits : public UnpackedInstTraits<GetOutputBlocksTraits>
 
   static String UnpackedAsPython(Array<String> outputs, String block_rv) {
     PythonAPICall py("get_output_blocks");
-    py.Input("block", block_rv);
+    py.Input("scope_block", block_rv);
```

Comment:
I'm not sure what the purpose of this change. Maybe a typo?

Reply:
Removed.